### PR TITLE
Component/MediaCard

### PR DESCRIPTION
**Pull request by miph1309**

### CODE REVIEW

File: src/components/MediaCard/MediaCard.vue [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    div(:class="infoContainerClassName")
+      Box(:padding="500")
+        BlockStack(:gap="200")
+          InlineStack(
+            align="space-between",
+            :wrap="false",
+            :gap="200",
+          )
+            Text(
+              v-if="typeof title === 'string'",
```
Comment (juzser) [inline src/components/MediaCard/MediaCard.vue:15]: It's not correct, it should be `v-if="hasSlot(slots.title)"` => display slot
else, using title prop. 
Check other components for more information

**Review by juzser — CHANGES_REQUESTED** [review]

File: src/components/MediaCard/MediaCard.vue [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+const infoContainerClassName = computed(() =>
+  classNames(
+    styles.InfoContainer,
+    props.portrait && styles.portrait,
+    props.size === 'small' && styles.sizeSmall,
+  ),
+);
+
+const actionClassName = classNames(
```
Comment (juzser) [inline src/components/MediaCard/MediaCard.vue:147]: computed

File: src/components/MediaCard/MediaCard.vue [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+                    :items="popoverActions",
+                    @action-any-item="togglePopoverActive",
+                  )
+                Button(
+                  v-if="attrs['onDismiss']",
+                  size="slim",
+                  variant="tertiary",
+                  :icon="XIcon"
+                  :accessibilityLabel="i18n.translate('Polaris.MediaCard.dismissButton')",
+                  @click="attrs['onDismiss']",
```
Comment (haild173094) [inline src/components/MediaCard/MediaCard.vue:53]: should it be `@click="emits('dismiss')"` :thinking:

File: src/components/MediaCard/MediaCard.vue [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+              position="absolute",
+              :insetInlineEnd="500",
+              :zIndex="'var(--p-z-index-2)'",
+            )
+              InlineStack(:gap="100", :wrap="false")
+                Popover(
+                  v-if="popoverActions.length > 0",
+                  :active="popoverActive",
+                  :onClose="togglePopoverActive",
+                  preferredAlignment="left"
```
Comment (haild173094) [inline src/components/MediaCard/MediaCard.vue:31]: `,`

File: src/components/MediaCard/MediaCard.vue [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+                  preferredPosition="below"
```
Comment (haild173094) [inline src/components/MediaCard/MediaCard.vue:32]: `,`

File: src/components/MediaCard/MediaCard.vue [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+        BlockStack(:gap="200")
+          InlineStack(
+            align="space-between",
+            :wrap="false",
+            :gap="200",
+          )
+            Text(
+              v-if="typeof title === 'string'",
+              variant="headingSm",
+              as="h2"
```
Comment (haild173094) [inline src/components/MediaCard/MediaCard.vue:17]: `,`

**Review by haild173094 — CHANGES_REQUESTED** [review]

**Review by juzser — APPROVED** [review]

**Pull request merged**